src/FormMigration/Steps/EmailSettings.php

```diff
@@ -20,7 +20,11 @@ public function process()
                 'status' => $notification->get_notification_status($this->formV2->id),
                 'email_subject' => $notification->get_email_subject($this->formV2->id),
                 'email_header' => $notification->get_email_header($this->formV2->id),
-                'email_message' => $notification->get_email_message($this->formV2->id),
+                'email_message' => str_replace(
+                    ['"“', '“"', '“'],
```

Comment:
This replacement is not working as expected. I migrated a P2P form and when opened it I got a fatal error that prevented the form preview from being rendered. Checking it deeper I realize that the JSON used in the Form Builder after the migration still keeps the same wrong quotes. 

What worked for me in my local tests was this replacement:

```
'email_message' => str_replace(
	['”', '”'],
	'"',
	$notification->get_email_message($this->formV2->id)
),
```

![image](https://github.com/impress-org/givewp/assets/16308864/45852348-26bb-464c-a04a-6a1973e107ac)

Reply:
I've found that I wasn't including all possible variations for the quotes. Please, check it again. It must work as expected now.